native/src/components/base/Icon.tsx

```diff
@@ -13,6 +13,7 @@ type IconProps = {
   directionDependent?: boolean
   reverse?: boolean
   style?: { width?: number; height?: number; color?: ColorValue }
+  color?: ColorValue
```

Comment:
🤔 I don't think this is used anywhere, is it?
```suggestion
```

Reply:
It's used by the `HeaderIcon` in the `CustomHeaderButtons`. We need it for hiding the search and language icon on the header in e.g. the settings route.